### DISCUSSION TITLE
hi, nice project can you share some where the .exe or installer? I would like to try in in my Windows 11, thansk

### DISCUSSION THREAD

**Discussion by CarlosJoseChaconChavarria** [cricketthomas/AzureKeyVaultExplorer#67] [General]
hi, nice project can you share some where the .exe or installer? I would like to try in in my Windows 11, thansk

**Comment by cricketthomas** [comment]
# **The app is now available on the microsoft store:**
https://apps.microsoft.com/detail/9mz794c6t74m?rtc=1&hl=en-us&gl=US&cid=github&gl=US




## The instructions below are for installing without the store:
Hey, thanks for checking it out! There's an exe available on this [page](https://github.com/cricketthomas/AzureKeyVaultExplorer/releases/tag/v0.0.236), just download and extract this file "[keyvaultexplorer.win-x64.tar](https://github.com/cricketthomas/AzureKeyVaultExplorer/releases/download/v0.0.236/keyvaultexplorer.win-x64.tar)". Once downloaded and extracted, the app's exe will be called keyvaultexplorerdesktop.exe.

![image](https://github.com/cricketthomas/AzureKeyVaultExplorer/assets/15821271/1e2eab02-3a05-4282-a242-058a3b995aee)

Right click on it, and select "properties" and then check the "unblock" checkbox. Then click "ok"/ "Apply". This is needed since it was built using GH actions on a different machine so windows security will block it. 
![image](https://github.com/cricketthomas/AzureKeyVaultExplorer/assets/15821271/35df6386-725a-40be-b822-9ba761b5fa23)

The double click the app. The app will launch and your browser window will immediately launch and prompt you to sign in with a Microsoft account. Right now, I am working on becoming a Microsoft partner so that my app doesn't say "unverified", but once youre logged in, you can navigate back to the app and start using it. 

I am working on future versions, with more improvements, such as editing of the secrets, better warning messages and more. 
Feedback is welcome! 

Video tutorial:
https://imgur.com/a/luCw3Th

**Reply by cricketthomas** [reply]
The app is now available on the Microsoft store!
